core/kazoo_auth/src/kz_auth_client.erl

```diff
@@ -304,3 +315,56 @@ add_claims(Claims) ->
 
 add_claim(<<"iat">>) -> {iat, kz_time:current_unix_tstamp()-500};
 add_claim(<<"exp">>) -> {exp, kz_time:current_unix_tstamp()+(2 * ?MILLISECONDS_IN_SECOND)}.
+
+-spec request(kz_term:ne_binary(), map()) -> kz_http:ret().
+request(URL, Token) ->
+    request('get', URL, <<>>, Token).
+
+-spec request(atom(), kz_term:ne_binary(), binary(), map()) -> {ok|error, any(), kz_term:proplist()}.
+request(Verb, URL, Body, #{token := #{authorization := Authorization}}) ->
+    Options = [{headers_as_is, true}, {ssl, [{versions, ['tlsv1.2']}]}],
+    {ok,{_,_, Host, _, _, _}} = http_uri:parse(URL),
+    Headers = [{<<"host">>, Host}
```

Comment:
can we standardize the headers? You have two keys as binaries and one as a string, `"host"` is all lower case while the others use capital letters.

Reply:
@jamesaimonetti that's intended (  `{headers_as_is, true}` )